samples/cpp/chat_sample/chat_sample.cpp

```diff
@@ -10,9 +12,37 @@ int main(int argc, char* argv[]) try {
     std::string prompt;
     std::string models_path = argv[1];
 
+    std::string model_path = models_path + "/openvino_model.xml";
+    std::string weights_path = std::regex_replace(model_path, std::regex(".xml"), ".bin");
+    std::ifstream model_file(model_path, std::ios::binary | std::ios::ate);
+    std::ifstream weights_file(weights_path, std::ios::binary | std::ios::ate);
```

Comment:
let's create a dedicated samples, which demonstrates that models are loaded via some "decrypt_llm_model", "decrypt_tokenizer", "decrypt_detokenizer" functions?

Reply:
This sample was added by mistake. Of course for reading from buffer i will crease a separate sample.